parquet/src/main/java/org/apache/iceberg/parquet/BaseColumnIterator.java

```diff
@@ -34,6 +34,7 @@
   protected long triplesRead = 0L;
   protected long advanceNextPageCount = 0L;
   protected Dictionary dictionary;
+  protected long rowPosition;
```

Comment:
Is this needed? I don't see any uses.

Reply:
Right, this and `setRowPosition` are no longer needed.